src/main/java/nextstep/qna/domain/DeleteHistory.java

```diff
@@ -19,13 +22,21 @@ public class DeleteHistory {
     public DeleteHistory() {
     }
 
-    public DeleteHistory(ContentType contentType, Long contentId, NsUser deletedBy, LocalDateTime createdDate) {
+    private DeleteHistory(ContentType contentType, Long contentId, NsUser deletedBy, LocalDateTime createdDate) {
         this.contentType = contentType;
         this.contentId = contentId;
         this.deletedBy = deletedBy;
         this.createdDate = createdDate;
     }
 
+    public static DeleteHistory questionDeleteHistory(Long contentId, NsUser deletedBy) {
```

Comment:
```suggestion
    public static DeleteHistory questionOf(Long contentId, NsUser deletedBy) {
```

호출하는곳에서는 `DeleteHistory.questionOf()` 이렇게 사용될텐데 
그렇다면 충분히 의미전달이 되지않을까? 생각이되어요~